queries2db.py

```diff
@@ -6,7 +6,7 @@
 
 def backup_queries():
 
-    cmd = "pg_dump -h localhost -p 5432 -U postgres -W  --table='queries' " \ 
+    cmd = "pg_dump -h localhost -p 5432 -U postgres -W  --table='queries' " \
```

Comment:
Is this a change?

Reply:
oh i guess maybe this is just an issue on my end, it has a space before, just delete a space